tests/PhpUnits/Helpers/HelperTest.php

```diff
@@ -575,7 +575,7 @@ public function testExplodeStrToArrSuccess() {
 					"Matt",
 					"Doe",
 				],
-				Helper::ExplodeStrToArr("John Matt Doe", " ")
+				Helper::ExplodeStrToArr("", "John Matt Doe")

```

Comment:
Careful here... u removed the space from the separator, and the tests are failing